docs/developer-guide/get-started/emt-building-howto.md

```diff
@@ -109,24 +106,23 @@ The resulting image will include the set of all `rpms` specified within the arra
 Note that you can only add the packages for which SPEC files exist. To add `nano` as an
 alternative text editor to the image:
 
-1. Define a new JSON file.
+1. Use the script to create a custom package list(e.g., `utilities.json`) and update an existing `imageconfig` JSON file(e.g., `edge-image.json`).
```

Comment:
```suggestion
1. Use the script to create a custom `utilities.json` file with a package list and update an existing `imageconfig` JSON file, for example, `edge-image.json`.
```

Reply:
@sgolebiewski-intel 
Is the "utilities.json" a fixed thing or just an example? That is not the same!

Reply:
No, it's not a fixed thing. The user can use any name for the JSON file. Thanks for catching that!